src/domain/config/repositories/ConfigRepository.ts

```diff
@@ -6,6 +6,7 @@ export interface ConfigRepositoryConstructor {
 }
```

Comment:
True, in fact I do want to have a ``Config`` entity, but it might come in future developments, I just used before implementation 🤣 

However @xurxodev might throw some light if these methods should be elsewhere.

Reply:
I have agreed with @tokland, config repository without a config entity it's a clue should not be a repository.